tiles/src/test/java/com/protomaps/basemap/layers/WaterTest.java

```diff
@@ -0,0 +1,24 @@
+package com.protomaps.basemap.layers;
+
+import static com.onthegomap.planetiler.TestUtils.newPolygon;
+
+import com.onthegomap.planetiler.reader.SimpleFeature;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class WaterTest extends LayerTest {
+  @Test
+  void simple() {
+    assertFeatures(15,
+      List.of(Map.of("pmap:kind", "other")),
```

Comment:
This smells off. Is this testing against a bug in the current code?

Reply:
No, I just threw in any odd example - changing it to `swimming_pool`